libsemigroups_pybind11/action.py

```diff
@@ -6,367 +6,356 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 
-# pylint:disable=no-name-in-module, unused-import, protected-access,
-# pylint:disable=missing-function-docstring
+# pylint:disable=no-name-in-module, unused-import,
+# pylint:disable=missing-function-docstring,
```

Comment:
There are a few unused imports in this file that don't get warned about due to these pylint suppressions. Since what we are now doing is more Pythonic than it was before, I think it's probably wise to remove the `pylint:disable` stuff and heed their warnings.

(This is the case for all of the python files in this PR, but I'll only write this comment once)

Reply:
I think I've fixed all of these now, should be good to go! Thanks @Joseph-Edwards